src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -60,9 +60,12 @@ public static partial class AppContext
         public static string BaseDirectory { get { return default(string); } }
         public static void SetSwitch(string switchName, bool isEnabled) { }
         public static event System.UnhandledExceptionEventHandler UnhandledException { add { } remove { } }
+        public static event System.EventHandler<System.Runtime.ExceptionServices.FirstChanceExceptionEventArgs> FirstChanceException { add { } remove { } }
```

Comment:
netstandard1.7 are APIs that we want to do for full framework. netcoreapp1.1 are .NET Core specific APIs. We have been adding all brand new APIs to netcoreapp1.1 (check recent commits in this file).